app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/DraftExciseMovementController.scala

```diff
@@ -141,7 +140,7 @@ class DraftExciseMovementController @Inject() (
     movementMessageService.saveNewMovement(newMovement).map {
       case Left(result)    =>
         auditService.auditMessage(message, "Failed to Save")
-        auditService.messageSubmittedNoMovement(message, true, submission.emcsCorrelationId, request)
+        auditService.messageSubmittedNoMovement(message, true, request.request.correlationId, request)
```

Comment:
EISSubmissionResponse no longer used, can be removed from method parameters?

Reply:
Will look to clean up